constants/process.js

```diff
@@ -0,0 +1,31 @@
+export default [
+  {
+    title: 'Volunteering',
+    subtitle: 'Innopolis University needs heroes like you to organize world-class events!',
+    img: 'images/landing/volunteering.png',
+    buttonLink: '#',
```

Comment:
And when it is a default link - no need to specify

Reply:
I put it there to make it easier to change when we put the actual link.
You will see it right in front of you without having to go back to the component and check the prop name and so on